whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/FullTextIndex.java

```diff
@@ -307,7 +307,31 @@ public RpslObject filterRpslObject(final RpslObject rpslObject) {
 
         return new RpslObject(rpslObject.getObjectId(), attributes);
     }
-    
+
+    public List<RpslAttribute> filterRpslAttributes(final Set<AttributeType> attributeType, final Map<String, Object> hitAttributes) {
```

Comment:
rename atributeType to attributeTypes  and attribute to attributeType?

Reply:
" and attribute to attributeType"
I the second statement of your comment. About what attribute are you talking about?